blocks/form/gdpr-confirmation.js

```diff
@@ -0,0 +1,90 @@
+function hasQueryParams() {
+  return !!window.location.search;
+}
+
+// Send a query parameters to the GDPR API Accept
+async function acceptResult(accept, urlProd, sucessRedirect, block) {
+  // get url parameters
+  const urlParams = new URLSearchParams(window.location.search);
+  const paramData = urlParams.get('data');
+  const paramInit = urlParams.get('init');
+  const url = urlProd + accept;
+  const options = {
+    method: 'PUT',
+    headers: { 'Content-type': 'application/json; charset=UTF-8' },
+    body: JSON.stringify({
+      data: paramData,
+      init: paramInit,
+      confirm: 'true',
+    }),
+  };
+  try {
+    const response = await fetch(url, options);
+    const resData = await response.text();
+    if (response.ok) {
+      window.open(sucessRedirect);
+    } else {
+      const { message } = JSON.parse(resData);
+      throw message;
+    }
+  } catch (error) {
+    const divError = document.createElement('div');
+    divError.className = 'diverror';
+    divError.innerHTML = 'Something went wrong. Try again';
```

Comment:
did you align with @DaphneKoren about how to show the error and what text to use for it? If we expect this to be multi-lingual we should use placeholders for texts, not sure if it is needed in this cases, please clarify

Reply:
Query added in gdpr channel waiting for clarification

Reply:
Current message and look is fine as it is simple and showing clear message as confirmation from Valeria Manenkova we go with cureent with as it is fine